### ISSUE TITLE
What would it take to expose your test helpers?

### ISSUE THREAD

**Issue by seanpdoyle** [mharris717/ember-drag-drop#11]
Since the `tests` directory isn't empty, I think it would be great to expose your suite of test helpers for developers to use to test drag-and-drop in their own apps.

What would it take to make your test helpers shareable?

Would documenting how they're used be a good start?

**Comment by csterritt** [comment]
Yeah, it would be great to have a way to test my app's drag and drop capability -- or at least, to test that the right action is called.

**Comment by mharris717** [comment]
I agree this would be helpful. I do a similar thing for ember-cli-pagination. Will try and make time to expose them. 

Thanks for taking the time to make an issue!

**Comment by mcfiredrill** [comment]
This seems like it should be possible when I read the docs, but I'm [having trouble importing the drag helper](https://github.com/mharris717/ember-drag-drop/issues/150).
https://github.com/mharris717/ember-drag-drop#test-helpers
